### PR TITLE
[WIP][FIX] OWEditDomain: Fix crash when variable's attributes are represented as…

### PR DESCRIPTION
**Pull request by VesnaT**
… lists

Fixes https://github.com/biolab/orange3/issues/1527

### CODE REVIEW

File: Orange/widgets/data/oweditdomain.py [hunk 1]
```diff
@@ -51,13 +51,13 @@ def variable_description(var):
                 var_type.__name__,
                 var.name,
                 (("values", tuple(var.values)),),
-                tuple(sorted(var.attributes.items())))
+                tuple((k, tuple(v)) for k, v in sorted(var.attributes.items())))
```
Comment (astaric) [inline Orange/widgets/data/oweditdomain.py]: What will happen if a variable has an attribute that has an int-value?

**Comment by janezd** [comment]
EditDomain is not the only widget that crashes in this way. Perhaps it would be better if we decide that all Variable's values must be hashable and fix the coloring to use tuples, instead of fixing all other widgets.

**Comment by kernc** [comment]
What Variable's values are those? Variable's attributes' keys?

**Comment by janezd** [comment]
@VesnaT, @astaric, #1552 also fixed the problem in `get_dict`, right? Can we close this PR?

**Comment by astaric** [comment]
The tests from this PR have been cherry-picked to #1552, the rest was solved in a different way.

**Pull request closed without merging**